templates/generic.php

```diff
@@ -15,11 +16,14 @@
 $totalImpuestosRetenidos = $comprobante->searchAttribute('cfdi:Impuestos', 'TotalImpuestosRetenidos');
 $conceptos = $comprobante->searchNodes('cfdi:Conceptos', 'cfdi:Concepto');
 $pagos = $comprobante->searchNodes('cfdi:Complemento', 'pago10:Pagos', 'pago10:Pago');
+$informacionGlobal = $comprobante->searchNode('cfdi:InformacionGlobal');
 $conceptoCounter = 0;
 $conceptoCount = $conceptos->count();
 $pagoCounter = 0;
 $pagoCount = $pagos->count();
-
+if (! isset($catalogos)) {
+    $catalagos = new \PhpCfdi\CfdiToPdf\CfdiCatalogs();
```

Comment:
```suggestion
    $catalogos = new \PhpCfdi\CfdiToPdf\CfdiCatalogs();
```

The typo is present in multiple places throughout this file.

Reply:
Fixed at 2022-09-22